addons/main/functions/VehicleAction/fnc_doVehicleSuppress.sqf

```diff
@@ -18,13 +18,13 @@
 params ["_unit", ["_pos", [0, 0, 0]]];
 
 private _vehicle = vehicle _unit;
-_pos = _pos call CBA_fnc_getPos;
+private _pos = _pos call CBA_fnc_getPos;
```

Comment:
`_pos` is already privated as it it part of params

Reply:
Good catch